yellowbrick/features/jointplot.py

```diff
@@ -275,18 +277,18 @@ def draw_xy(self, X, y, **kwargs):
         if self.xy_args is None:
             self.xy_args = {}
 
-        facecolor_x = self.xy_args.pop("facecolor_x", "#dddddd")
+        facecolor_x = self.xy_args.pop("facecolor_x", FACECOLOR)
         self.x_ax.set_facecolor(facecolor_x)
-        facecolor_y = self.xy_args.pop("facecolor_y", "#dddddd")
+        facecolor_y = self.xy_args.pop("facecolor_y", FACECOLOR)
         self.y_ax.set_facecolor(facecolor_y)
 
 
         if self.xy_plot == "hist":
             hist_bins = self.xy_args.pop("bins", 50)
             self.xy_args.setdefault("alpha", 0.4)
-            histcolor_x = self.xy_args.pop("histcolor_x", "#6897bb")
+            histcolor_x = self.xy_args.pop("histcolor_x", HISTCOLOR)
             self.x_ax.set_facecolor(facecolor_x)
-            histcolor_y = self.xy_args.pop("histcolor_y", "#6897bb")
+            histcolor_y = self.xy_args.pop("histcolor_y", HISTCOLOR)
```

Comment:
Trying to make the color a constant so it could be changed in one place.

Reply:
👍